app/lua/luac_cross/Makefile

```diff
@@ -76,11 +77,12 @@ endif
 
 $(ODIR)/%.o: %.c
 	@mkdir -p $(ODIR);
+	$(summary) HOSTCC $(CURDIR)/$<
 	$(CC) $(if $(findstring $<,$(DSRCS)),$(DFLAGS),$(CFLAGS)) $(COPTS_$(*F)) -o $@ -c $<
 
 $(ODIR)/%.d: %.c
 	@mkdir -p $(ODIR);
-	@echo DEPEND: $(CC) -M $(CFLAGS) $<
+	$(summary) DEPEND: HOSTCC $(CURDIR)/$<
```

Comment:
This change breaks the make if run standalone as HOSTCC lost HOSTLD aren't defined.